tutorials/en_us/JARS/ADVANCED.md

```diff
@@ -2,20 +2,24 @@
 
 If you want to go a little more in-depth with JAR picking, you've come to the right place.
 
-## Most popular Spigot/Bukkit JARs are :
+## Most popular Spigot/Bukkit JARs (PLUGINS):
+
+### Recommended top picks:
 
 - [PaperMC](https://papermc.io/) - Most popular JAR for Spigot/Bukkit, quite fast and stable, they roll out small patches at least every 2 days or so, good choice if you're just starting with your MC server and don't have much past experience because it's fast and simple. Most if not all spigot plugins work with this JAR.
   starting with your MC server and don't have much past experience because it's fast and simple.
-- [Spigot](https://www.spigotmc.org/) - Original fork of Bukkit, improved performance vs CraftBukkit, still widely used, seems to be harder for newbies to download because it links to Jenkins,
```

Comment:
we should link to [getbukkit](https://getbukkit.org) then where people can download the jars more easy.